modules/ROOT/pages/performance-and-impact.adoc

```diff
@@ -10,11 +10,61 @@ Based on typical usage, the expected CPU utilization is less than 5%. For more i
 
 Anypoint Monitoring asynchronously collects data, which does not block Mule runtime data processing.
 
+=== Gold and Platinum Subscriptions
+
+Gold and Platinum subscribers to Anypoint Platform receive modest data retention and feature limits:
+
+Logs::
+Log limits are documented in xref:runtime-manager::viewing-log-data.adoc#log-persistence[Runtime Manager].
+
+Application Metrics::
+30 days
+
+Enhanced API Metrics::
+30 days. The limit for data in the *Client Applications* tab is 1 hour.
+
+Basic Alerts::
+50 basic alerts
+
+////
+Gold and Platinum subscribers to Anypoint Platform receive
 * Minute Resolution for Metrics.
 * Gold and Platinum subscriptions:
  ** API analytics metrics data are retained for 90 days.
  ** Application metrics data are retained for 30 days.
+////
+
+=== Titanium Subscriptions
+
+Titanium subscribers to Anypoint Platform receive more robust data retention and feature limits:
+
+Logs::
+In raw format, your log limit is equal to the storage space that you receive with your Titanium subscription.
```

Comment:
I wouldn't use definitive word like "equal to". How about something like "Storage: The storage space you receive with your Titanium subscription is used to store both logs, metrics data, and search indexes. As indexes and metrics do not take up much storage capacity, you can expect majority of the allocated storage will be used to store the raw log data."

Reply:
```suggestion
In raw format, your logs can occupy as much space as you receive with your Titanium subscription. Note that the storage space you receive with your Titanium subscription is used to store logs, metrics data, and search indexes. As indexes and metrics do not take up much storage capacity, the majority of the allocated storage can be used to store the raw log data.
```